com.jetbrains.CLion.yml

```diff
@@ -0,0 +1,63 @@
+app-id: com.jetbrains.CLion
+branch: stable
+command: clion
+runtime: org.freedesktop.Sdk
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+
+modules:   
+  - udev-175.json
+  
+  - name: clion
+    buildsystem: simple
+    build-commands: 
+      - install -D apply_extra /app/bin/apply_extra
+      - install -D clion.sh /app/bin/clion
+      - install -Dm644 com.jetbrains.CLion.desktop /app/share/applications/com.jetbrains.CLion.desktop
+      - install -Dm644 com.jetbrains.CLion.png /app/share/icons/hicolor/128x128/apps/com.jetbrains.CLion.png
+      - install -Dm644 com.jetbrains.CLion-64.png /app/share/icons/hicolor/64x64/apps/com.jetbrains.CLion.png
+      - mkdir -p /app/lib/i386-linux-gnu /app/lib/debug/lib/i386-linux-gnu
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands: 
+          - tar xf clion.tar.gz
+          - rm clion.tar.gz
+          - mv clion-* clion
+
+      - type: script
+        dest-filename: clion.sh
+        commands: 
+          - exec env PATH=$PATH:/var/data/python/bin /app/extra/clion/bin/clion.sh 
+
+      - type: file
+        path: com.jetbrains.CLion.desktop
+      
+      - type: file
+        path: com.jetbrains.CLion.png
+      
+      - type: file
+        path: com.jetbrains.CLion-64.png
+
+      - type: extra-data
```

Comment:
extra-data must use the flatpak data checker, see other jetbrains apps here on Flathub.

Reply:
I made the updates for x-checker-data, hopefully it will build this time. Thanks